src/sql/parts/notebook/outputs/sanitizer.ts

```diff
@@ -0,0 +1,1054 @@
+
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+//import sanitizeHtml = require('typings/modules/sanitize-html/index');
```

Comment:
Please remove the commented-out import.

Reply:
Done and double checking all the files for cleanup.